code/modules/mob/living/simple_animal/hostile/syndicate.dm

```diff
@@ -133,9 +127,8 @@
 	flying = 1
 	speak_emote = list("states")
 	gold_core_spawnable = 1
+	del_on_death = 1
 
-/mob/living/simple_animal/hostile/viscerator/death(gibbed)
-	..(gibbed)
-	visible_message("<span class='danger'><b>[src]</b> is smashed into pieces!</span>")
-	qdel(src)
-	return
+/mob/living/simple_animal/hostile/viscerator/New()
+	..()
+	deathmessage = "<b>[src]</b> is smashed into pieces!"

```

Comment:
I think it would be easier to remove the src from the deathmessage text and add the src manually in the visible_message() (since basically all messages start with [src]). i.e. `visible_message("<spanshit>[src] [deathmessage]</span>"`